partcad/src/partcad/part_factory_brep.py

```
@@ -0,0 +1,107 @@
+                )
+                sys.stderr.write(errors)
+
+                response = base64.b64decode(response_serialized)
+                result = pickle.loads(response)
+
+                if not result["success"]:
+                    pc_logging.error(result["exception"])
+                    raise Exception(result["exception"])
+                shape = result["shape"]
+            else:
+                time.sleep(0.0001)
```

Ruff (0.8.2)

Async functions should not call `time.sleep`

(ASYNC251)